.clusterfuzzlite/build.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+###############################################################################
+#        _       _             _   _            _____  _
+#       | |     | |           | | | |          |  __ \(_)
+#       | | ___ | |__  _ __   | |_| |__   ___  | |__) |_ _ __  _ __   ___ _ __
+#   _   | |/ _ \| '_ \| '_ \  | __| '_ \ / _ \ |  _  /| | '_ \| '_ \ / _ \ '__|
+#  | |__| | (_) | | | | | | | | |_| | | |  __/ | | \ \| | |_) | |_) |  __/ |
+#   \____/ \___/|_| |_|_| |_|  \__|_| |_|\___| |_|  \_\_| .__/| .__/ \___|_|
+#                                                       | |   | |
+#                                                       |_|   |_|
+#
+# Copyright (c) 2023 Claudio André <claudioandre.br at gmail.com>
+#
+# This program comes with ABSOLUTELY NO WARRANTY; express or implied.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, as expressed in version 2, seen at
+# http://www.gnu.org/licenses/gpl-2.0.html
+###############################################################################
+# Script for building John the Ripper (for fuzzing)
+# More info at https://github.com/openwall/john-packages
+
+cd /upstream/src || exit 1
+
+./configure --enable-asan --enable-ubsan --enable-fuzz
+make -sj4
```

Comment:
To use libFuzzer properly, we should do something like this (for each fuzz target).

```sh
./configure --enable-libfuzzer
make clean && make wpapcap2john  # && ./wpapcap2john => copy to $OUT, the execution is done by the action.
```